docs/content/flink/sql-ddl.md

```diff
@@ -97,26 +99,26 @@ Also, you can create [FlinkGenericCatalog]({{< ref "flink/quick-start" >}}).
 > metastore.create.as.acid=false
 > ```
 
-#### Synchronizing Partitions into Hive Metastore
+##### Synchronizing Partitions into Hive Metastore
```

Comment:
The hierarchy is too deep, I think it's better not to need this affiliation.

Reply:
Well, I just keep this level the same as next.

Reply:
I already updated it.please take a look,thank you!